packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -480,6 +486,8 @@ export class AmqpConnection {
       rpcOptions,
       channel,
     });
+
+    return consumerTag;
```

Comment:
My general recommendation is to return an object so that in the future if we decide to export more we can easily just add a new property

Reply:
@underfisk Good call, updated [here](https://github.com/golevelup/nestjs/pull/472/commits/1597daa6bd3de78b89eb5b98821eaeaf084d300c).

Are you able to approve the pending workflow, to give the added e2e tests a shot? (I'm having some difficulty spinning them up locally, they seem to be wired primarily for CI use?)

Reply:
@craigotis CI approved :)